docs/configuration-options.md

```diff
@@ -94,6 +92,50 @@ values:
 - `discovered`: enqueue a job only for newly discovered repositories
 - `disabled`: No jobs are enqueued
 
+### Job Scheduling Options
+
+> [!IMPORTANT]  
+> Job scheduling options are different between Community Edition and Enterprise Edition.
+> 
+> **Renovate Enterprise Edition** allows job scheduling to be customized so that active repos run more frequently and stale repos run less often.
+> The Enterprise job schedulers are:
+> - `MEND_RNV_CRON_JOB_SCHEDULER_HOT` (Default Hourly - Active repos: new, activated)
+> - `MEND_RNV_CRON_JOB_SCHEDULER_COLD` (Default Daily - Semi-active repos: onboarded, onboarding, failed)
+> - `MEND_RNV_CRON_JOB_SCHEDULER_CAPPED` (Default Weekly - Blocked repos: resource-limit, timeout)
+> - `MEND_RNV_CRON_JOB_SCHEDULER_ALL` (Default Monthly - All enabled repos: not disabled)
+> 
+> **Renovate Community Edition** has a single job scheduler that runs against all repos, regardless of their repo state.
+> - `MEND_RNV_CRON_JOB_SCHEDULER_ALL` (Default Hourly - All repos)
+> 
+> See below for details
+
+> [!Note]
+> 
+> `MEND_RNV_CRON_JOB_SCHEDULER` is deprecated from v7.3.0.
+> - For Renovate Community Edition: use `MEND_RNV_CRON_JOB_SCHEDULER_ALL` (see below)
+> - For Renovate Enterprise Edition: use `MEND_RNV_CRON_JOB_SCHEDULER_HOT` (see below)
+
+**`MEND_RNV_CRON_JOB_SCHEDULER_HOT`**: [Enterprise Only] Runs all activate and new repositories. Defaults to hourly (0 * * * *)
+  * Runs repos with status: `new`, `activated`
+ 
+Note: An `activated` repository is one that has onboarded and also accepted at least one Renovate PR.
+
+Note: This option overrides the deprecated `MEND_RNV_CRON_JOB_SCHEDULER` flag.
+
+**`MEND_RNV_CRON_JOB_SCHEDULER_COLD`**: [Enterprise Only] Runs all semi-active repositories. Defaults to daily (10 0 * * *)
+* Runs repos with status: `onboarded`, `onboarding`, `failed`
+
+**`MEND_RNV_CRON_JOB_SCHEDULER_CAPPED`**: [Enterprise Only] Runs all repositories that are blocked. Defaults to weekly (20 0 * * 0)
+* Runs repos with status: `resource-limit`, `timeout`
```

Comment:
It would be nice with an explanation of `resource-limit` as well, how does Renovate figure out this (I mean, in contrast to a `timeout`)? How can we users monitor this?

Since these are configs we can change, it is important to know what is actually the implications of which repos are affected :-)